_posts/2025-11-26-security-releases.md

```diff
@@ -0,0 +1,34 @@
+---
+title: November 2025 Security Releases
+description: Security release for body-parser has been published. We recommend that all users upgrade as soon as possible.
+tags: security vulnerabilities
+authors:
+  - name: Ulises Gascón
+    github: UlisesGascon
+---
+
+The Express team has released a new patch version of [body-parser](https://www.npmjs.com/package/body-parser) addressing a moderate-severity security vulnerability.
+
+
+{% include admonitions/warning.html 
+content="We recommend upgrading to the latest version of body-parser immediately to secure your applications."
```

Comment:
```suggestion
content="We recommend upgrading to the latest version of body-parser to secure your applications."
```

WDYT? As this vulnerability is a moderate one... maybe we can recommend to upgrade but not "immediately"?